components/prism-nevod.js

```diff
@@ -0,0 +1,79 @@
+Prism.languages.nevod = {
+	'comment': /(?:\/\/.*)|(?:\/\*[\s\S]*?(?:\*\/|$))/,
+	'string': [
+		{
+			pattern: /"(?:""|[^"])*"(?!")!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		},
+		{
+			pattern: /'(?:''|[^'])*'(?!')!?\*?/,
+			greedy: true,
+			inside: {
+				'modifiers': /!$|!\*$|\*$/,
+			},
+		}
+	],
+	'keyword': /@(?:inside|outside|having|search|where)\b/,
```

Comment:
I would like to suggest the following to make the whole grammar a lot simpler: Match keywords after patterns that include them.

I other languages, we use a design pattern that roughly works like this:

```js
{
	// string, comments, whatever
	
	'function': {
		pattern: /(\bfunction\s+)\w+/,
		lookbehind: true
	},
	'class-name': {
		pattern: /(\bclass\s+)\w+/,
		lookbehind: true
	},
	'keyword': /\b(?:function|class|some|other|keywords)\b/

	// other tokens that don't need/include keywords
}
``` 

This has the advantage that tokens that are _defined by_ keywords don't have to tokenize these keywords, so there is only one `keyword` pattern. (You see this design pattern being used in [Ruby](https://github.com/PrismJS/prism/blob/a68f1fb6e23143ada9676fa6f66af72a3b1eebe8/components/prism-ruby.js#L17).)

Reply:
Fixed as suggested.